src/subtitles-octopus.js

```diff
@@ -649,14 +673,43 @@ var SubtitlesOctopus = function (options) {
         }
     };
 
+    function _computeCanvasSize(width, height) {
+        if (self.prescaleTradeoff > 1) {
+            if (height * self.prescaleTradeoff <= self.softHeightLimit) {
+                width *= self.prescaleTradeoff;
+                height *= self.prescaleTradeoff;
+            } else if (height < self.softHeightLimit) {
+                width = width * self.softHeightLimit / height;
+                height = self.softHeightLimit;
+            } else if (height >= self.hardHeightLimit) {
+                width = width * self.hardHeightLimit / height;
+                height = self.hardHeightLimit;
+            }
+        } else if (height >= self.softHeightLimit) {
```

Comment:
If prescaleTradeoff is exactly 1 we should probably not scale width and height with floats at all to get pixel perfect rendering. Or introduce a flag to disable this canvas size scaling altogether.

Reply:
added ability to pass in `null` meaning "disable scaling" (also defaulted to it)

Reply:
Ooh that's neat.